src/adapters/PeopleAdapter.js

```diff
@@ -6,11 +6,26 @@ import {throwError} from 'rxjs';
  * adhering to the exact parameters and structure of the returned objects
  */
 export default class PeopleAdapter {
+  /**
+   * A Person object with details about the person
+   * @typedef {object} PersonObject
+   * @param {string} id The person identifier
+   * @param {Array.<String>} emails An array of emails for the person
```

Comment:
This would still be `string`, no?

```suggestion
   * @param {Array.<string>} emails An array of emails for the person
```